core-reactive/build.gradle

```diff
@@ -14,6 +14,11 @@ dependencies {
 tasks.named("compileKotlin") {
     kotlinOptions {
         jvmTarget = 17
-        languageVersion = "1.7"
+    }
+}
+
+tasks.named("compileTestKotlin") {
```

Comment:
This should be done in a common plugin rather than copy/pasting into every build script